test/uk/gov/nationalarchives/omega/editorial/support/DateParserSpec.scala

```diff
@@ -83,6 +107,9 @@ class DateParserSpec extends WordSpec with MustMatchers {
         Seq("29/2/2022", "30/2/2022", "31/2/2022", "42/10/2022", "10/14/2022")
           .foreach(rawDate => parse(rawDate) mustBe empty)
       }
+      "given a negative year" in {
+        parse("1/11/-10") mustBe empty
+      }
     }
```

Comment:
Should we have a scenario for when the year is zero?

Reply:
Done

Reply:
Confirmed.